### PR TITLE
Add unit test

### PR DESCRIPTION
**Pull request by ZhaoQi99**

### CODE REVIEW

**Comment by ZhaoQi99** [comment]
close #5

File: test/test_campus.py [hunk 1]
```diff
@@ -42,6 +41,31 @@ def test_get_list_2(self):
...
+        self.assertTrue(test_result['success'])
+        self.assertIsInstance(test_result['data'], bytes)
+
+    @unittest.skipIf(consumption_status != 200, "状态码不等于200，就跳过该测试")
+    def test_get_photo_2(self):
+        test = Campus('201608714')
+        test_result = test.get_photo()
+        self.assertIsInstance(test_result, dict)
+        self.assertEqual(test_result['msg'] , '获取成功')
+        self.assertTrue(test_result['success'], True)
```
Comment (ZhaoQi99) [inline test/test_campus.py]: 状态码不是200，这里应该是`False`吧.

File: test/test_campus.py [hunk 2]
```diff
@@ -42,6 +41,31 @@ def test_get_list_2(self):
...
+
+        test = Campus('xxxxxx')
+        test_result = test.get_photo()
+        self.assertIsInstance(test_result, dict)
+        self.assertEqual(test_result['msg'], '获取成功')
+        self.assertTrue(test_result['success'],True)
```
Comment (ZhaoQi99) [inline test/test_campus.py]: 同上...

File: test/test_room.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import unittest
+import urllib.request
+from urllib.error import URLError
+
+import snnusdk.room as room
```
Comment (ZhaoQi99) [inline test/test_room.py:5]: 这块为啥要重复import

File: test/test_room.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        test_result = test.get_all_rooms()
+        self.assertIsInstance(test_result, list)
+
+    @unittest.skipIf(Rooms_status != 200, "状态码不等于200，就跳过该测试")
+    def test_query_one_room(self):
+        test = Room(3, '雁塔教学八楼')
+        test_result = test.query_one_room('8101')
+        self.assertIsInstance(test_result, dict)
+        with self.assertRaises(RoomNotFoundError):
+            test.query_one_room('8014')
```
Comment (ZhaoQi99) [inline test/test_room.py]: 这个写成`self.assertRaises(RoomNotFoundError,test.query_one_room,'8014')`吧..

File: test/test_urp.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+    def setUp(self):
+        try:
+            Urp_status = urllib.request.urlopen(
+                url=Urp.URLs.HOST, timeout=5).code
+        except URLError:
+            Urp_status = -1
+
+        try:
+            Urp_status = urllib.request.urlopen(
```
Comment (ZhaoQi99) [inline test/test_urp.py]: 这个光测一下URP其实就行吧...

File: test/test_urp.py [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+    @unittest.skipIf(Urp_status != 200, "状态码不等于200，就跳过该测试")
+    def test_get_course(self):
+        test = Urp('41612123', 'jjy19980415')
+        test_result = test.get_courses()
+        self.assertIsInstance(test_result , list)
+
+        test = Urp('xxx', 'xxx')
+        with self.assertRaises(UnauthorizedError):
+            test.get_courses()
```
Comment (ZhaoQi99) [inline test/test_urp.py]: 同上写法...

**Comment by xiekeyi98** [comment]
This pull request **introduces 2 alerts** when merging bbd03f7a15be78085c745d755618aea2da4d8b21 into aa49d7db61aec3c8cd2e7199ab38966074306b13 - [view on LGTM.com](https://lgtm.com/projects/g/snnucs/SNNU-SDK/rev/pr-934a284271b90e806827d3ca635501041f14dfb1)

**new alerts:**

* 1 for Wrong number of arguments in a call
* 1 for Wrong number of arguments in a class instantiation

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: test/test_urp.py [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+
+
+
```
Comment (ZhaoQi99) [inline test/test_urp.py]: 咋这么多空行...

File: test/test_News.py [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+from snnusdk.configs import SPIDER_CONFIG as links
+
+class TestNews(unittest.TestCase):
+    code_status = 200
+
+    def setUp(self):
+        try:
+            for link in links:
+                code_status = urllib.request.urlopen(url=link['url'], timeout=5).code
+                if code_status != 200:
```
Comment (ZhaoQi99) [inline test/test_News.py]: 这样写只要有一个url无法访问，就炸了....

**Pull request merged**